validator/lib/validateTileset.js

```diff
@@ -173,3 +184,103 @@ function validateTileHierarchy(root, tilesetDirectory) {
             return message;
         });
 }
+
+function checkBoundingVolume(tileBV, parentBV, tileTransform, parentTransform) {
+    var returnString = undefined;
+
+    if (defined(tileBV) && defined(parentBV) && (defined(tileBV.box) || defined(tileBV.sphere) || defined(tileBV.region)) && (defined(parentBV.box) || defined(parentBV.sphere) || defined(parentBV.region))) {
+        if (defined(tileBV.box) && defined(parentBV.box)) {
+            // Box in Box check
+            var transformed_tileBox = getTransformedBox(tileBV.box, tileTransform);
+            var transformed_parentBox = getTransformedBox(parentBV.box, parentTransform);
+            if (boxInsideBox(transformed_tileBox, transformed_parentBox) !== true) {
+                returnString = 'box [' + tileBV.box + '] is not within box [' + parentBV.box + ']';
+                return returnString;
+            } else {
+                return returnString;
+            }
```

Comment:
All these returns could be simplified. Just set the `returnString`, and at the end of `checkBoundingVolume` return `returnString`. Also `message` is a better name than `returnString`.